cmd/itzo-launcher/itzo-launcher.go

```diff
@@ -121,17 +132,24 @@ func RunItzo(itzoPath string) error {
 	return nil
 }
 
-func RunAddons() error {
+func readCellConfig() (map[string]string, error) {
 	config := make(map[string]string)
 	contents, err := ioutil.ReadFile(CellConfigFile)
 	if err != nil {
 		klog.Warningf("reading %s: %v", CellConfigFile, err)
+		return config, err
 	} else {
 		err = yaml.Unmarshal(contents, &config)
 		if err != nil {
 			klog.Warningf("unmarshaling config %s: %v", contents, err)
+			return config, err
```

Comment:
nit: cant we return nil rather than the config if we error out?

Reply:
:+1: good point! Changed to nil and added handling nil config in few other places